integration-tests/smoke/keeper_test.go

```diff
@@ -97,7 +97,7 @@ var highBCPTRegistryConfig = contracts.KeeperRegistrySettings{
 	FallbackLinkPrice:    big.NewInt(2e18),
 }
 
-var _ = Describe("Keeper Suite @keeper", func() {
+var _ = PDescribe("Keeper Suite @keeper", func() {
```

Comment:
Did you mean to leave this as suite as pending?

Reply:
Fixed